integration/terraform/ecs/linux/README.md

```diff
@@ -1,7 +1,9 @@
 Running ECS Fargate Integration Tests
 =========================
+## Prerequisite
+* ECR Repository with the docker image
```

Comment:
Do we have any instructions on how to set up the ECR repository?

Reply:
We do [have the instructions](https://docs.aws.amazon.com/AmazonECR/latest/userguide/getting-started-console.html)